src/Auth/ReadFromTextFileCredentials.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace YdbPlatform\Ydb\Auth;
+
+class ReadFromTextFileCredentials extends Auth
+{
+
+    protected $fileName;
+    protected $readInterval;
+
+    /**
+     * @param string $fileName
+     * @param int $readInterval
+     */
+    public function __construct(string $fileName = "token.json", int $readInterval = 600)
+    {
+        if(!file_exists($fileName)){
+            throw new \Exception("File $fileName is not exists");
+        }
+        $this->fileName = $fileName;
+        $this->readInterval = $readInterval;
+    }
+
+    public function getTokenInfo(): TokenInfo
+    {
+        if(!file_exists($this->fileName)){
```

Comment:
May be many reason why file_get_contents can't get file content, for example bad permissions.

It is better - read file content, then check result with `=== false` and throw exception for any error, not file not exists only.